locales-uk-UA.xml

```diff
@@ -16,50 +16,41 @@
     <date-part name="year"/>
   </date>
   <terms>
-    <term name="accessed">accessed</term>
+    <term name="accessed">дата звернення</term>
     <term name="and">і</term>
     <term name="and others">та інші</term>
     <term name="anonymous">анонімний</term>
     <term name="anonymous" form="short">анон.</term>
     <term name="at">на</term>
-    <term name="available at">available at</term>
-    <term name="by">by</term>
-    <term name="circa">circa</term>
+    <term name="available at">доступний у</term>
+    <term name="by">відповідно до:</term>
```

Comment:
> `<term name="by">відповідно до:</term>`

Is the ":" here part of the translation, or separating punctuation that follows the translation of "by". If the latter, this colon should be moved to the CSL style. In this locale file, typically the only punctuation used should be periods for abbreviations.